blog.md

```diff
@@ -111,38 +186,52 @@ To git@github.com:paulczar/ci-demo.git
+1. Delete the `ephemeral` environment for Pull Request as it is no longer needed.
+2. Create and deploy a new release of the contents of the `development` branch to the `staging` environment in Deis (http://stage-ghost.ci-demo.paulczar.net).
 
-![IMAGE JENKINS JOB](http://)
+![IMAGE JENKINS JOB](https://ci-demo-ghost-images.s3.amazonaws.com/2015/Nov/ci_staging_deploy-1448031350720.png)
 
-Originally when I started building this demo I had assumed that being able to perform actions on PR merges/closes would be simple, but I quickly discovered none of the CI tools support performing actions on PR close. Thankfully I was able to find a useful [blog](http://chloky.com/github-json-payload-in-jenkins/) post that described how to set up a custom job with a webhook that could process the github payload.
+![](https://ci-demo-ghost-images.s3.amazonaws.com/2015/Nov/stage_ghost-1448031723495.png)
 
+_Originally when I started building this demo I had assumed that being able to perform actions on PR merges/closes would be simple, but I quickly discovered that __none of the CI tools, that I could find, supported performing actions on PR close__. Thankfully I was able to find a useful [blog](http://chloky.com/github-json-payload-in-jenkins/) post that described how to set up a custom job with a webhook that could process the GitHub payload._
 
 #### Production
 
-An application `ghost` lives on the DEIS PAAS and DEIS is configured to point it at a production database hosted on the clustered Percona database via the DB loadbalancer.
+Promoting the build from `staging` to `production` is a two step process:
+
+1. The user who wishes to promote it creates a pull request from the development branch to the master branch. Jenkins will see this and kick off some final tests.
+
+![](https://ci-demo-ghost-images.s3.amazonaws.com/2015/Nov/PR_to_master-1448031582932.png)
+
+2. Another user then has to merge that pull request which will fire off a Jenkins job to push the code to Deis which cuts a new release and deploys it to the `production` environment (http://ghost.ci-demo.paulczar.net).
+
+![](https://ci-demo-ghost-images.s3.amazonaws.com/2015/Nov/ci_prod_deploy-1448031629520.png)
+
+## Conclusion
+
+Coming from an Operationss background I though that figuring out how to build and run a PaaS from the metal up would be a really interesting learning exercise. It was, but I didn't expect to discover that actually running an application on that PaaS and figuring out the development workflow and CI/CD pipeline is far more interesting.
+
+Building and running an application and CI/CD pipeline on top of the PaaS I built placed me in the shoes of the developers whom I support. It demonstrated to me that my argument of needing to move the cloud user experience from infrastructure to application is indeed a valid argument.
```

Comment:
I love this content, but it needs to be massaged a bit. Crucially, you need to clearly state *why* this exercise validated your hypothesis (right now the reader is just taking your word for it, heh). Also, and perhaps I'm projecting here, but it seems like you're talking about *empathy* here - if so, please say that. We need more of it in the tech world. :smile: 

Consider:
« The most interesting outcome of this exercise, however, is *increased empathy*: the process of building and using this platform placed me directly in the shoes of the very developers I support. It further demonstrated that by moving the user experience from to the application layer [PUT A BRIEF BUT COMPELLING ARGUMENT HERE]. »